app/views/layouts/admin/_custom_colors.html.slim

```diff
@@ -7,7 +7,13 @@
 
 style type="text/css"
 
-  | /* General links */
+
+  | /* For customizing */
+    .custom_dark_accent {
+      color: #{colors[:accent_darkened]};
+    }
+    
```

Comment:
these had been merged before, and got lost, which is why some things were gray instead of red as in the mocks here: https://redmine.sassafras.coop/attachments/791

Reply:
Hmm, strange. Is it worth doing some forensic work to see what happened or no?

Reply:
yeah, i found the commit where it got lost. i was manually merging a release branch that's outside of our normal release flow, back in early feb.  So getting our release process smoothed out like we're talking about on wed next wk will help make that not happen again.